src/atoms/Spacer/index.tsx

```diff
@@ -1,7 +1,62 @@
+import React from 'react';
 import styled from 'styled-components';
 
 import { withSpacing } from '../../util';
 
-export default styled.div`
+const Spacer = styled.div`
   ${withSpacing};
 `;
+
+export default Spacer;
+
+export const Spacer4: React.SFC = () => (
+  <Spacer mr={[1, 1, 1, 1, 1]} mb={[1, 1, 1, 1, 1]} />
+);
+
+export const Spacer8: React.SFC = () => (
+  <Spacer mr={[1, 1, 2, 2, 2]} mb={[1, 1, 2, 2, 2]} />
+);
+
+export const Spacer12: React.SFC = () => (
+  <Spacer mr={[2, 2, 2, 3, 3]} mb={[2, 2, 2, 3, 3]} />
+);
+
+export const Spacer16: React.SFC = () => (
+  <Spacer mr={[2, 2, 3, 4, 4]} mb={[2, 2, 3, 4, 4]} />
+);
+
+export const Spacer20: React.SFC = () => (
+  <Spacer mr={[3, 3, 4, 5, 5]} mb={[3, 3, 4, 5, 5]} />
+);
+
+export const Spacer24: React.SFC = () => (
+  <Spacer mr={[4, 4, 5, 6, 6]} mb={[4, 4, 5, 6, 6]} />
+);
+
+export const Spacer32: React.SFC = () => (
+  <Spacer mr={[6, 6, 7, 8, 8]} mb={[6, 6, 7, 8, 8]} />
+);
+
+export const Spacer40: React.SFC = () => (
+  <Spacer mr={[5, 5, 6, 10, 10]} mb={[5, 5, 6, 10, 10]} />
```

Comment:
Can I just check this is the right way round with spacer 32.  The reason I ask is every other spacer is bigger than the previous but this one is smaller than the last on smaller devices

Reply:
good catch, I mixed something up there. Let me fix that